x-pack/plugins/remote_clusters/public/shim.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { npSetup, npStart } from 'ui/new_platform';
+import { management, MANAGEMENT_BREADCRUMB } from 'ui/management';
+import { fatalError } from 'ui/notify';
+import routes from 'ui/routes';
+import { DOC_LINK_VERSION, ELASTIC_WEBSITE_URL } from 'ui/documentation_links';
+
+import { trackUiMetric as track } from '../../../../src/legacy/core_plugins/ui_metric/public';
+
+export function createShim() {
+  const {
+    core: { chrome },
+  } = npSetup;
+
+  const {
+    core: { i18n, notifications, http, injectedMetadata },
+  } = npStart;
+
+  return {
+    coreStart: {
+      chrome,
+      i18n,
+      notifications: {
+        ...notifications,
+        fatalError,
+      },
+      injectedMetadata,
+      http,
+      documentation: {
+        elasticWebsiteUrl: ELASTIC_WEBSITE_URL,
+        docLinkVersion: DOC_LINK_VERSION,
+      },
+      routes: {
+        register: routes.when.bind(routes),
+      },
+    },
+    pluginsStart: {
+      management: {
```

Comment:
@lukeelmers We've shimmed the Management plugin's `start` contract. I thought you might be interested in what we depend upon and/or possibly want to provide us some feedback.